backend/core/src/app.module.ts

```diff
@@ -40,7 +40,27 @@ import { CatchAllFilter } from "./shared/filters/catch-all-filter"
 
 export function applicationSetup(app: INestApplication) {
   const { httpAdapter } = app.get(HttpAdapterHost)
-  const allowList = process.env.CORS_ORIGINS || []
+
+  // It isn't usually possible to pass in an array via env var, so we'll turn
+  // a string into an array when needed
+
+  // Default to empty array
+  let allowList = []
+
+  // If CORS_ORIGINS is set...
+  if (process.env.CORS_ORIGINS) {
+    // use it directly if it's an array
+    if (Array.isArray(process.env.CORS_ORIGINS)) {
+      allowList = process.env.CORS_ORIGINS
+      // otherwise split on "," if it's a string
+    } else if (typeof process.env.CORS_ORIGINS == "string") {
+      allowList = process.env.CORS_ORIGINS.split(",")
```

Comment:
will it cause problems if the CSV format includes a space after the comma, like `"domain, domain2"`, which will cause the second value to be `" domain2"`?

Reply:
I thought about adding a `trim()` to each value but decided against it.  I'll go ahead and add it back in